pages/api/notify.tsx

```diff
@@ -0,0 +1,47 @@
+import formidable from 'formidable';
+import {Input, Telegram} from 'telegraf';
+
+import type {NextApiRequest, NextApiResponse} from 'next';
+
+
+type TRequest = {
+	body: {
+		screenshot?: string;
+		messages?: string;
+	};
+} & NextApiRequest
+
+export default async function handler(req: TRequest, res: NextApiResponse<boolean>): Promise<void> {
+	const telegram = new Telegram(process.env.TELEGRAM_BOT as string);
+	try {
+		const form = formidable();
+		const formData = await new Promise(async (resolve, reject): Promise<void> => {
+			form.parse(req, async (err: Error, fields: unknown, files: unknown): Promise<void> => {
+				if (err) {
+					reject('error');
+				}
+				resolve({fields, files});
+			});
+		});
+		const {fields, files} = await formData as any;
+		const {screenshot} = files;
+		await telegram.sendPhoto(
+			process.env.TELEGRAM_CHAT as string,
+			Input.fromLocalFile(screenshot.filepath),
+			{
+				caption: fields.messages,
+				parse_mode: 'Markdown'
+			}
+		);
```

Comment:
2 minor things:

* We could improve the types for the `formData`, check if the suggestion below makes sense;
* `process.env.TELEGRAM_CHAT` can be `undefined`, so better to check for that instead of casting;

```suggestion
		const formData = await new Promise<{fields: TRequest['body'], files: TRequest['body']}>(async (resolve, reject): Promise<void> => {
			form.parse(req, async (err: Error, fields: TRequest['body'], files: TRequest['body']): Promise<void> => {
				if (err) {
					reject('error');
				}
				resolve({fields, files});
			});
		});
		const {fields, files: {screenshot}} = formData;
		if (process.env.TELEGRAM_CHAT) {
			await telegram.sendPhoto(
				process.env.TELEGRAM_CHAT,
				Input.fromLocalFile(screenshot.filepath),
				{
					caption: fields.messages,
					parse_mode: 'Markdown'
				}
			);
		}
```